<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiImExportAssistant</class>
 <widget class="QWizard" name="UiImExportAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>545</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Im- / Export Assistant</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::NoCancelButtonOnLastPage</set>
  </property>
  <widget class="QWizardPage" name="wizardPageIntroduction">
   <property name="title">
    <string>Introducing</string>
   </property>
   <property name="subTitle">
    <string>Welcome to the OlbaFlinx Import &amp; Export assistant</string>
   </property>
   <layout class="QVBoxLayout" name="vlIntroduction">
    <item>
     <widget class="QLabel" name="lblIntroducingTop">
      <property name="text">
       <string>The Import / Export Assistant guides you through several steps to import and/or export your transactions.

Which step do you want to perform?</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="vsIntroductionTop">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlIntroductionImport">
      <item>
       <widget class="QRadioButton" name="rbIntroductionImport">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Import</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblIntroductionImportHelp">
        <property name="text">
         <string>Imports various formats that are supported by AqBanking</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlIntroductionExport">
      <item>
       <widget class="QRadioButton" name="rbIntroductionExport">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblIntroductionExportHelp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Exports to various formats supported by AqBanking</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="vsIntroductionMiddle">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="lblIntroducingBottom">
      <property name="text">
       <string>In the next step, the corresponding profiles are loaded depending on which option was selected.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="vsIntroductionBottom">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPageImExport">
   <property name="title">
    <string>Im- / Export</string>
   </property>
   <property name="subTitle">
    <string>Select one of the loaded profiles and the file to import and or export</string>
   </property>
   <layout class="QVBoxLayout" name="vlImExport">
    <item>
     <layout class="QHBoxLayout" name="hlImExportPlugin">
      <item>
       <widget class="QLabel" name="lblImExportPlugin">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Plugin</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxImExportPlugin"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlImExportProfile">
      <item>
       <widget class="QLabel" name="lblImExportProfile">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Profile</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxImExportProfile"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="vsImExportProfile">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlImExportProfileHelp">
      <item>
       <widget class="QLabel" name="lblImExportProfileHhelp">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblImExportProfileHelpDesc">
        <property name="text">
         <string>---</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="vsImExportMiddle">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlImExportFile">
      <item>
       <widget class="QLabel" name="lblImExportFile">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>File</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leImExportFile"/>
      </item>
      <item>
       <widget class="QToolButton" name="tbImExportFile">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="vlImExportBottom">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>94</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QProgressBar" name="pbImExportProgress">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
